src/detectors/FHCAL/FHCAL.cc

```diff
@@ -363,10 +363,20 @@ extern "C" {
 
         app->Add(
           new JOmniFactoryGeneratorT<TrackClusterMergeSplitter_factory>(
-            "LFHCALSplitMergeProtoClusters",
-            {"LFHCALIslandProtoClusters",
-             "CalorimeterTrackProjections"},
-            {"LFHCALSplitMergeProtoClusters"},
+            "LFHCALSplitMergeClustersWithoutShapes",
+            {"LFHCALClusters",
+             "CalorimeterTrackProjections",
+             "LFHCALClusterAssociations",
+#if EDM4EIC_VERSION_MAJOR >= 7
+             "LFHCALRawHitAssocitions"},
+#else
+             "LFHCALHit"},
+#endif
+            {"LFHCALSplitMergeClustersWithoutShapes",
+#if EDM4EIC_VERSION_MAJOR >= 8
+             "LFHCALTrackSplitMergeClusterMatches",
```

Comment:
```suggestion
             "LFHCALSplitMergeClusterMatches",
```
naming consistency here and elsewhere?